### ISSUE TITLE
[Bug] Cannot run procedural simulation for more than 6 days.

### ISSUE THREAD

**Issue by ArnoBen** [elonlit/Genesis#19]
I am trying to create an earth-like simulation so I obviously chose Genesis for this purpose.
However, I tried to run the simulation for 7 days and couldn't get past day 6.
Any idea where this problem comes from ?

**Comment by leoheck** [comment]
Can you at least start with a population smaller than 2?

**Comment by leoheck** [comment]
It looks like a simulation cannot be 7 since the scheduler uses the 7th day to do some rest work. This is how this thing was built I suppose.

**Comment by ArnoBen** [comment]
@leoheck Yes, I can start with a population of 1 as well, the two items are actually very similar because the second item is simply a slight variation of the first one (I'm trying something funny here, we'll see how it goes).

Guess I'll have to pause the simulation every 7th day, no big deal. 

Thank you for your answer, I'll update if I run into more issues. In the meantime, I'll keep a close eye to my algogen pool development.

**Comment by leoheck** [comment]
I see. Now, you don't have to stop the simulation _every_ 7 days. But you have to do that on the first 7th day. After that, your scheduler can have this soft deadline where it is going to stop or reduce the load on the next 7th day, but it may not be able to do so, which is ok and also expected behavior. Even with some people saying, we are not going to see a flood of issues after. It is highly improbable that this flood is going to happen.
